pyproject.toml

```
@@ -42,3 +42,53 @@ maintainers = [{name="CIL developers", email="tomography+cil@stfc.ac.uk"}]
 requires-python = ">=3.10"
 readme = "README.md"
 keywords = ["tomography", "optimisation"]
+dependencies = [
+    #"cil-data>=22", # missing from PyPI
+    "dxchange",
+    "h5py",
+    #"ipp==2021.12.*", # PyPI conflicts with conda package
+    "ipywidgets",
+    "numba",
+    "numpy>=1.23",
```

```suggestion
    "numpy>=1.23,<2.0",
```
And the second I hit approve, I noticed this. Aren't there numpy 2.0 support issues at the moment?

---

It's not currently explicitly `<2` anywhere else in the codebase atm, so best to leave #1915 for later.